install/gcp/README.md

```diff
@@ -12,7 +12,15 @@
 - Give it the **Project Owner** role
 - Select key type **JSON**
 - Click **Create**
-- Download the credential file to `~/.config/gcloud/terraform.json`
+- Download the credential file
+
+## Configuration
+
+### Environment Variables
+
+- `GOOGLE_CREDENTIALS` (path or contents of the credentials file)
+- `GOOGLE_PROJECT` (required)
```

Comment:
It might be worth noting that it is the ID of the project and not the name. I have made this mistake before and it looks like you just have some permissions problem when you try to apply the template